app/demo-loop/Transporter.cc

```diff
@@ -211,10 +212,13 @@ Transporter<M>::Transporter(TransporterInput inp) : input_(std::move(inp))
 template<MemSpace M>
 TransporterResult Transporter<M>::operator()(const TrackInitParams& primaries)
 {
+    CELER_LOG(status) << "Initializing primaries";
+    Stopwatch get_transport_time;
     // Initialize results
     TransporterResult result;
     result.time.steps.reserve(input_.max_steps);
     result.initializers.reserve(input_.max_steps);
+    result.active.reserve(input_.max_steps);
```

Comment:
In the case where we don't specify `max_steps`, this would result in a really large malloc, no?

Reply:
Whoops! Didn't even notice that consequence. Yes, thank you: I'll wrap that in a conditional.